app-devel/pixi/autobuild/beyond

```diff
@@ -0,0 +1,9 @@
+abinfo "Installing shell completions ..."
+
+"$SRCDIR"/target/release/pixi completion --shell bash > "$SRCDIR"/pixi.bash
```

Comment:
Optimise this a bit, please - create all the directories first, then pipe them straight to their place.